python/ray/train/predictor.py

```diff
@@ -179,21 +197,21 @@ def _predict_pandas(self, data: "pd.DataFrame", **kwargs) -> "pd.DataFrame":
         raise NotImplementedError
 
     @DeveloperAPI
-    def _predict_arrow(self, data: "pyarrow.Table", **kwargs) -> "pyarrow.Table":
-        """Perform inference on an Arrow Table.
+    def _predict_numpy(
+        self, data: Union[np.ndarray, Dict[str, np.ndarray]], **kwargs
+    ) -> Union[np.ndarray, Dict[str, np.ndarray]]:
+        """Perform inference on a Numpy data.
 
-        Predictors can implement this method instead of ``_predict_pandas``
-        for better performance when the input batch type is a Numpy array, dict of
-        numpy arrays, or an Arrow Table as conversion from these types are zero copy.
+        All DL base predictors (ex: Torch/TF) are expected to implement this method.
```

Comment:
I don't think we necessarily need to make this claim. As long as one of `_predict_pandas` or `_predict_numpy` is implemented, then that should be fine.